src/Backend/Core/Engine/TemplateModifiers.php

```diff
@@ -272,4 +272,39 @@ public static function truncate($var = null, $length, $useHellip = true, $closes
             return \SpoonFilter::htmlspecialchars($var, ENT_QUOTES);
         }
     }
+
+    /**
+     * Shows a v or x to indicate the boolean state (Y|N, j|n, true|false)
+     *
+     * @param string|bool $status
+     * @param bool        $reverse show the opposite of the status
+     * @return string
+     */
+    public static function showBool($status, $reverse = false)
+    {
+        $showTrue = '<strong style="color:green">&#10003;</strong>';
```

Comment:
Why a strong-element? wouldn't it be better to use a `<span>`-element with a classname. So the colors can be decided by the designer/frontender.